templates/components/multiagent/typescript/workflow/type.ts

```diff
@@ -1,11 +1,36 @@
 import { WorkflowEvent } from "@llamaindex/core/workflow";
+import { MessageContent } from "llamaindex";
 
 export type AgentInput = {
-  message: string;
+  message: string | MessageContent;
```

Comment:
TBD

Reply:
In TS, I see the message is passed with annotations which is a object, not just a string

Reply:
Ah, the `MessageContent` type is also a string so we can simplify this: `message: MessageContent`